schema/artist/index.js

```diff
@@ -103,11 +111,34 @@ const ArtistType = new GraphQLObjectType({
       deathday: {
         type: GraphQLString,
       },
+      biography: {
+        type: Article.type,
+        description: 'The Artist biography article written by Artsy',
+        resolve: ({ _id }) => {
+          return positron('articles', {
+            published: true,
+            biography_for_artist_id: _id,
+            limit: 1,
+          }).then(articles => first(articles.results));
+        },
+      },
       alternate_names: {
         type: new GraphQLList(GraphQLString),
       },
       meta: Meta,
-      blurb: markdown(),
+      blurb: assign({}, markdown(), {
+        resolve: (artist, args, ast) => {
+          const blurb = markdown().resolve(artist, args, ast);
+          if (blurb.length) {
+            return blurb;
+          }
+          return gravity(`artist/${artist.id}/partner_artists`, {
+            size: 1,
+            sort: '-published_artworks_count',
+            has_biography: true,
+          }).then(({ biography }) => biography);
+        },
+      }),
```

Comment:
OK I see now what's happening. But some weirdness still: `markdown` is gluing a `format` arg onto this, which isn't going to do anything if there's no blurb; thus falling back to the biography. I'd just export the formatter in the markdown field file and use it directly; since you probably want to format the biography as well?